src/Generators/Microsoft.Gen.Logging/Parsing/AttributeProcessors.cs

```diff
@@ -180,4 +189,7 @@ public static (bool omitReferenceName, ITypeSymbol providerType, string provider
 
         return (omitReferenceName, providerType!, providerMethodName!);
     }
+
+    public static string ExtractTagNameAttributeValues(AttributeData attr)
+        => attr.ConstructorArguments[0].Value as string ?? string.Empty;
```

Comment:
What if it doesn't have ctor's arguments? E.g., when syntax is invalid

Reply:
We never get here in that case. Roslyn simply won't list attribute which it couldn't construct.